src/bundleList.js

```diff
@@ -6,10 +6,7 @@ const path = require('path');
 const downloadFileSync = require('download-file-sync');
 const _ = require('lodash');
 
-const peliasConfig = require('pelias-config').generate();
-
-// validate the WOF importer configuration before continuing
-require('./configValidation').validate(peliasConfig.imports.whosonfirst);
+const peliasConfig = require( 'pelias-config' ).generate(require('../schema'));
```

Comment:
I don't think you need to validate against the schema here since this is only used in download_data.js and import.js which already validate.